src/buffer/out/TextAttribute.hpp

```diff
@@ -161,6 +164,13 @@ class TextAttribute final
     {
         return WI_IsAnyFlagSet(_wAttrLegacy, COMMON_LVB_GRID_HORIZONTAL | COMMON_LVB_GRID_LVERTICAL | COMMON_LVB_GRID_RVERTICAL | COMMON_LVB_UNDERSCORE);
     }
+    constexpr bool HasAnyExtendedAttributes() const noexcept
+    {
+        return GetExtendedAttributes() != ExtendedAttributes::Normal ||

```

Comment:
(fwiw: since `GetExtendedAttributes` doesn't do any translation or synthesis on its own, why should we use it instead of just using `_extendedAttrs` directly?)